tests/snuba/api/endpoints/test_organization_events_v2.py

```diff
@@ -0,0 +1,152 @@
+from __future__ import absolute_import
+
+
+from datetime import timedelta
+from django.utils import timezone
+from django.core.urlresolvers import reverse
+
+from sentry.models import Group
+from sentry.testutils import APITestCase, SnubaTestCase
+
+
+class OrganizationEventsTestBase(APITestCase, SnubaTestCase):
+    def setUp(self):
+        super(OrganizationEventsTestBase, self).setUp()
+        self.min_ago = (timezone.now() - timedelta(minutes=1)).isoformat()[:19]
```

Comment:
Does using `timezone.now()` protect your tests from complications with time anomalies (i.e. daylight savings and things like that)? One alternative, I've seen in other tests to using the current time is to use a hardcoded set time in the past.

Reply:
IIRC it can cause problems if the hardcoded time is before the retention period so would prefer to go with a relative time